package.json

```diff
@@ -47,7 +47,9 @@
     "crypto-browserify": "^3.12.0",
     "framer-motion": "^6.3.15",
     "immer": "^9.0.15",
+    "ion-js": "^4.3.0",
```

Comment:
How come?

Reply:
oh i was trying to use ion-js before but i think we don't need this anymore